workspaces/liferay-sample-workspace/client-extensions/liferay-sample-global-css/webpack.config.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const path = require('path');
+const webpack = require('webpack');
+
+const DEVELOPMENT = process.env.NODE_ENV === 'development';
+
+module.exports = {
+	devtool: DEVELOPMENT ? 'source-map' : false,
+	entry: {
+		global: './assets/global.css',
+	},
+	mode: DEVELOPMENT ? 'development' : 'production',
+	module: {
+		rules: [
+			{
+				test: /\.css$/i,
+				use: [MiniCssExtractPlugin.loader, 'css-loader'],
+			},
+		],
+	},
+	optimization: {
+		minimize: !DEVELOPMENT,
+	},
+	output: {
+		path: path.resolve('static'),
+	},
+	plugins: [
+		new MiniCssExtractPlugin({
+			filename: '[name].[contenthash].css',
+		}),
```

Comment:
Just a couple of thoughts about this plugin:
* I've seen it generates `js` files. Probably they don't hurt as they're not added to the bundle but they can be a source of misinterpretations. We can use a couple of plugins to delete it, `RemoveEmptyScriptsPlugin` and  `FileManagerPlugin`
* On subsequent builds, unless dev cleans them, plugin will keep adding stuff to `static/` and so more than one asset will be packaged. This makes CX `url` field to concatenate all of them, producing a wrong URL. Unless workspace provides something at the gradle level to clean these things, seems this script should handle the situation and keep only 1 asset

Reply:
I considered adding the plugin but didn't want to add more deps given that it's not really necessary because we are not adding the .js files to the ZIP.

About the double build issue, it happens in all examples, not just this one, so I've created a thread in Slack to ask the Dev Tools team -> https://liferay.slack.com/archives/CL08WH5V0/p1709191517142209

Once we decide what to do we can create an LPD to tackle it outside of the context of this PR.

Reply:
Since @dsanz has found this in webpack's documentation:

> In general it's good practice to clean the /dist folder before each build, so that only used files will be generated. Let's take care of that with output.clean option.
(source: https://webpack.js.org/guides/output-management/#cleaning-up-the-dist-folder)

I'm gonna try adding it to all our samples to prevent multiple files from appearing in the build folders and then, if the Workspace team can think of any better solution we may revisit it.

Reply:
I've commited the changes to the `webpack.config.js` files.